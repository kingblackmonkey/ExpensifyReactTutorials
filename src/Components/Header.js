import React from 'react';
import {NavLink} from 'react-router-dom';
const Header= ()=>{
  
  
    return (
   
    <header>
        <h1>Expense App</h1>
        <div>
            <NavLink  to="/"  exact = {true}  activeClassName="selected" >               
               DashBorad           
            </NavLink>
        </div>
       
        <div>
            <NavLink to={`/addExpense`} activeClassName="selected">           
               
         Add Expense
                 
            </NavLink>
        </div>
     
       
            
    </header>
  
    )
  }

  export default Header;