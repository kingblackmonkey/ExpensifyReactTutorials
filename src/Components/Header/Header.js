import React from "react"
export const Header = (props)=>
    (
    <div className = 'header'> 
       <h1 className= 'header__title' style = {props.style}>{props.title}</h1>
       <h4 className="header__sub-title">{props.subTitle}</h4>       
     </div>
   
   )


Header.defaultProps = {title: "Big APPPP"};