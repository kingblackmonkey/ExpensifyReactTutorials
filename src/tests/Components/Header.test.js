
import React from 'react';
import { shallow} from 'enzyme';
import Header from '../../Components/Header'

const wrapper = shallow(<Header />);


test ('should render header component', ()=>{
    expect (wrapper).toMatchSnapshot()
    
})