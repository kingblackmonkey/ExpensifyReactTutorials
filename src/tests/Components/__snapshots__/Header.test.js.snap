// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header component 1`] = `
<header>
  <h1>
    Expense App
  </h1>
  <div>
    <NavLink
      activeClassName="selected"
      exact={true}
      to="/"
    >
      DashBorad
    </NavLink>
  </div>
  <div>
    <NavLink
      activeClassName="selected"
      to="/addExpense"
    >
      Add Expense
    </NavLink>
  </div>
</header>
`;
